airflow_blogpost.txt

```diff
@@ -0,0 +1,110 @@
+
+Versatile Data Kit allows users to develop and manage workloads called Data jobs. These jobs can be entirely SQL code, entirely Python code, or a mix of both, and can be run either locally, or be deployed to a Kubernetes environment and be managed by the VDK Control Service. Deployed jobs can be scheduled to run periodically using a cron-like scheduling format.
+For more information on Versatile Data Kit reference the official documentation here -
+https://github.com/vmware/versatile-data-kit/wiki/Introduction
+
+To integrate VDK with Apache Airflow, we've created the airflow-provider-vdk Python package which features two user-facing classes:
+A job-triggering operator, named VDKOperator, which can be used to trigger a job execution using a configured connection to an instance of the VDK Control Service, given a particular job name and job team. This operator can be set to run synchronously or asynchronously: in the first case, the operator will only complete after the job execution completes and only be considered successful if the job execution itself is successful; in the second case, the operator is considered successful if the job execution is successfully triggered with no consideration for whether the job passes successfully or not;
+A sensor, named VDKSensor, which pings an instance of the VDK Control Service to check whether a job execution has completed and is successful, given a particular job execution ID.
+
+Given this integration, a user can now instantiate an Airflow DAG where each workload in this DAG triggers a particular deployed Data job. This allows for the expression and management of Data job dependencies, which was previously not possible.
+
+If you haven't installed VDK already, you can do so by following the instructions found here: https://github.com/vmware/versatile-data-kit/wiki/Installation#install-sdk
+
+In short - install kind, helm, and docker; make sure you have a working Python environment running Python 3.7+ and run `pip install quickstart-vdk` and then `vdk server —install`.
+
+To install and set up a local instance of Airflow, follow the instructions found here: https://airflow.apache.org/docs/apache-airflow/stable/start/local.html
+
+Then, install the VDK provider for Airflow by running `pip install airflow-provider-vdk`.
+
+First, you'll need to configure your connection to the deployed VDK Control Service. This can be done through the CLI, as well as through the web UI. From the 'Admin' dropdown menu, open the 'Connections' tab and press the 'Add a new connection' button. From there, name your connection appropriately (we'll go with `vdk-conn` for our purposes), set the connection type to `VDK`, and if using default parameters for the VDK Control Service instantiation, set the host to 'localhost' and the port to 8092 (note that if you're using a different instance of the Control Service, you'll have to specify a different host and port).
```

Comment:
Here screenshots can help. 

Is there programatic way to do that (so we can simply make users copy the script)